ios/Classes/MapboxMapController.swift

```diff
@@ -400,7 +400,21 @@ class MapboxMapController: NSObject, FlutterPlatformView, MGLMapViewDelegate, Ma
         }
     }
     
-
+    func mapView(_ mapView: MGLMapView, regionWillChangeAnimated animated: Bool) {
+        channel?.invokeMethod("camera#onMoveStarted", arguments: []);
+    }
+    
+    func mapViewRegionIsChanging(_ mapView: MGLMapView) {
+        if !trackCameraPosition { return };
+        channel?.invokeMethod("camera#onMove", arguments: [
+            "position": getCamera()?.toDict(mapView: mapView)
+        ]);
+    }
+    
+    func mapViewDidBecomeIdle(_ mapView: MGLMapView) {
+        channel?.invokeMethod("camera#onIdle", arguments: []);
+    }
+    
```

Comment:
Works a charm, definitely needs to be used in the next update. Surprised wasn't implemented before.